force-app/main/default/classes/SOQL.cls

```diff
@@ -235,9 +317,16 @@ public with sharing class SOQL {
         Map<Id, String> accountIdsToNames = new Map<Id, String>();
 
         // Query the Accounts
+        List<Account> acctsNotInCountry =[SELECT Id, Name
+        FROM Account
+        WHERE BillingCountry != :country];
         // Loop through each Account and add the Id and Name to the map
+        for(Account tempAcct: acctsNotInCountry){
+            accountIdsToNames.put(tempAcct.Id, tempAcct.Name);
```

Comment:
Try to combine the for loop and the SOQL query in a SOQL For Loop to directly iterate over the results of the SOQL Query

Reply:
I will need to discuss this wih you one on one cause I I am not sure I can do this: